presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigUtils.java

```diff
@@ -59,16 +59,16 @@ public static File storeConfigFiles(Path configDir)
         return tempFile;
     }
 
-    public static void deployConfigFiles(File configTar, Path configDir)
+    public static void untarFile(File src, Path dest)
```

Comment:
This method should be in `AgentFileUtils`, and both arguments should be `Path`s.